NEWS

```diff
@@ -12,22 +12,21 @@ CHANGES WITH 235:
           bond0 interface using systemd-networkd.
 
         * systemd-analyze gained new verbs "get-log-level" and "get-log-target"
-          which print the logging level and target of the system manager,
-          respectively. They complement the existing "set-log-level" and
-          "set-log-target" verbs, which can be used to change those values.
+          which print the logging level and target of the system manager. They
+          complement the existing "set-log-level" and "set-log-target" verbs
+          used to change those values.
 
         * journald.conf gained a new boolean setting ReadKMsg= which defaults
           to on. If turned off kernel log messages will not be read by
-          systemd-journald and not be included in the logs. It also gained a
-          new setting LineMax= for configuring the maximum line length to allow
-          when converting STDOUT/STDERR log streams into individual log
-          records. The new default for this value is 48K, up from the previous
-          hardcoded 4K.
+          systemd-journald or included in the logs. It also gained a new
+          setting LineMax= for configuring the maximum line length in
+          STDOUT/STDERR log streams. The new default for this value is 48K, up
+          from the previous hardcoded 2048.
```

Comment:
Is the change of the documented old limit (4K → 2048) correct? (I tried looking at the old code, but couldn’t figure out where the old limit was applied.)

Reply:
```console
$ getconf LINE_MAX
2048
```

Reply:
Ah, thanks.